homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -3,11 +3,16 @@
 
 from homeassistant.components.device_tracker import DeviceScanner
 
-from . import DATA_ASUSWRT
+from .const import DATA_ASUSWRT
 
 _LOGGER = logging.getLogger(__name__)
 
 
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the asuswrt scanner."""
+    return
```

Comment:
We should consider using a separate target branch instead of merging this directly to dev branch if we don't want to include the new device tracker implementation in this PR. We prefer to not release a mix of legacy platform config and entry setup platforms.

Let me know and I'll create a copy of dev branch and push to this remote. Then you can rebase and swap target here in the PR. That way we have time to do multiple PRs before this change is released.

Reply:
I'm a little bit confused about this. As I said before I implemented about 1 month ago PR #42860 that also introduce ScannerEntity to replace legacy platform, but until now was ignored. For me is the same work on this or on the other, let me know how to proceed.

Reply:
By not merging directly to the dev branch we have more time than one release cycle to do multiple PRs to that other target branch. Then when we're ready to release, we do a PR from the other target branch to dev branch.

This allows us to divide the work in smaller PRs while still making sure everything is released in the same release.

Reply:
That's fine, so we can start with this PR and than move ahead with next step, creating multiple PR in new branch, correct?
So please go ahead with new branch, than I will rebase my PR.

Reply:
Great! :+1: